.env.example

```diff
@@ -1,2 +1,3 @@
 BUCKET_NAME_PREFIX="my-website-honestly"
 BUCKET_NAME_SUFFIX="dev"
+API_KEY='my-website-honestly'
```

Comment:
Ah, misunderstanding. It seems this is the name of the key, AWS handles generation for us.

No need for this to be an env var then,